src/cpl/nuopc/rof_import_export.F90

```diff
@@ -0,0 +1,679 @@
+module rof_import_export
+
```

Comment:
I'd like to see a little more header information, and description.

Reply:
done.